### DISCUSSION TITLE
Como puedo enviar un PBmessage con las respuestas de un selector coleccion

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#358] [2. Questions and Answers] [answered]
Hola lo explico aqui

En un formulario tengo una componente colección "Frutas", con: "Manzana", "Tomate", "Pera", "Manzana", "Palta" y "Ciruela", en la cual selecciono 3 frutas y las quiero mandar a un mensaje de texto: "Seleccionaste: Manzana, Tomate, Pera".

Mi pregunta es: cómo puedo hacer eso?

Se que con usar **iterate** + **nwr** + **Messages**, puedo mostrar 1 fruta a la vez pero no todas en un mensaje.

No sé si me serviría upgradear una colección "frutas seleccionadas", y en el done del iterate, colocar el campo frutas seleccionadas.

O algo asi creo

Plz gracias =>

**Comment by acedward** [comment, marked as answer]
Me imagino que estas usando algo así:
```
 "content": " $VALUE#answer|process|0"
```
si es así, inténtalo con el arreglo completo:
```
 "content": " $VALUE#answer|process|[toString=> ]"
```
https://doc.cotalker.com/docs/documentation/automation/admin_cotlang
![image](https://user-images.githubusercontent.com/3951895/128747040-e18edc09-59e0-487c-8a66-bf6e1c2676fa.png)

**Comment by NicolasPLG** [comment]
hola @acedward perfecto lo probaré como lo dices, dejaré cerrada la respuesta y si debo consultar de nuevo creo otra discusión con referencia a esta.

Gracas=>
